array_analyzer/load/debug_plots.py

```diff
@@ -39,10 +39,9 @@ def assign_region(target_, props_, intensity_image_=None):
 
     min_row, min_col, max_row, max_col = props_.bbox
     if intensity_image_ is None:
-        target_[min_row:max_row, min_col:max_col] = props_.intensity_image
+        target_[min_col:max_col, min_row:max_row] = props_.intensity_image
```

Comment:
NumPy indexing is (row, column) (which we often call (y, x)) so you'll probably need to change the min max values you get from props_.bbox to reflect numpy indexing.

Reply:
Admittedly i'm getting turned about here.  If we switch the row/col from props_.bbox, the change to image slicing effectively goes back to the old broken way.  Do you mean we should swap
```
min_row, min_col, max_row, max_col = props_.bbox
```
to
```
min_col, min_row, max_col, max_row= props_.bbox
```
and keep the slicing like before?:
```
target_[min_row:max_row, min_col:max_col] = props_.intensity_image
```

Reply:
Yes, that's what I meant. Thanks!